### ISSUE TITLE
[Suggestion] Diversity Decay Min and Max Values

### ISSUE THREAD

**Issue by WatameCraft** [CreativeMD/Spice-of-Life-Onion#28]
Previous iterations of Spice of Life had settings in their configs to help manipulate the minimum and maximum percent values allowed for diversity to decay. The range is 0.0 to 1.0 (0% - 100%).

Returning this setting would be great, especially for easily defining and calculating breakpoints in a player's food progress.

**Comment by CreativeMD** [comment]
I'm not sure what exactly this setting should do. Diversity "decays" in this version because you eat new food with lower diversity properties (either because you have eaten it already or if it just less diverse in general). Not sure how a min max setting could be implemented here. Maybe the old spice of life worked different? Or maybe I did not understand it correctly?

**Comment by CreativeMD** [comment]
I will close this due to inactivity. Feel free to respond here anyways.

**Comment by WatameCraft** [comment]
Woopsies, completely forgot about this. In older spice of life versions, you could make it so food would always give a minimum percent of their diversity pts based on this range. At 1.0 in particular, this would stop food from losing diversity points at all until it was booted off the list. That means that the porkchop with 4.0 diversity at the end of the list still gives 4.0 diversity. 

This made it incredibly easy and predictable to set up breakpoints in food diversity. I did not have to contend with any complicated, unseen formulas; whatever diversity value the food showed, that's how much it will always contribute. If I wanted a player to receive x buffs, I could reliably gate it off with specific combinations of endgame/high-maintenance food, without that food suddenly regressing behind the buff due to the decay.

**Comment by CreativeMD** [comment]
So you would like to have a minimum contribution level if the food has been eaten some time ago (till it is passed out of the trackCount which is 32 by default)?

I could also add an option which disables loosing it's diversity contribution when you eat new food. Basically simplifying the formula. Would that work for you?

Also sorry for not responding earlier.

**Comment by WatameCraft** [comment]
Yes, that would work. It would keep the values static so it would reduce the need for recalculations on both the modpack maker and especially the players.

**Comment by CreativeMD** [comment]
Added this feature in the newest version.
